userguide/tutorials/aws-single-ami.adoc

```diff
@@ -76,7 +76,7 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/sing
 
 You can now login to KAUI from your browser using the URL https://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as **Public IPV4 Address**. This should display the KAUI login screen. The default credentials are: *admin/{EC2 instance ID}*. The first few requests might be a bit slow as Kill Bill initializes itself.
 
-For an introduction to KAUI, see our https://docs.killbill.io/latest/getting_started.html#_using_kill_bill_with_kaui[Getting Started] guide.
+For an introduction to KAUI, see our https://docs.killbill.io/latest/quick_start_with_kaui.html[_Quick Start with Kaui_] guide.
```

Comment:
Wondering if linking to the complete [Kaui User Guide](https://docs.killbill.io/latest/userguide_kaui.html) would be better here? I'll let you decide :)

Reply:
Yes - changed it.